domains/voldy.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "broisvoldemort",
+           "email": "",
+           "discord": "1094541800032706631"
+        },
+    
+        "record": {
+            "A": ["128.204.223.119"]
```

Comment:
Gives a error, please give a preview of your website.

Reply:
Instead of an A record, make it a CNAME to voldy.ftp.sh

Reply:
you'd have to change the file though